.github/workflows/main.yml

```diff
@@ -97,6 +97,34 @@ jobs:
       rebuild-qa-containers: ${{ needs.init.outputs.rebuild-qa-containers == 'true' }}
     secrets: inherit
 
+  memory-checked-unit-tests:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/collector-builder:${{ needs.build-builder-image.outputs.collector-builder-tag }}
+    needs:
+      - build-builder-image
+    strategy:
+      fail-fast: false
+      matrix:
+        config-args:
+        - -DDISABLE_PROFILING=ON -DADDRESS_SANITIZER=ON
```

Comment:
Does `DISABLE_PROFILING=ON` means profiling is actually enabled?

Reply:
No, it means there's no profiling, (I hate double negative variables).